src/test/java/de/unistuttgart/iste/gits/courseservice/service/ResourceServiceTest.java

```diff
@@ -128,7 +122,7 @@ private ResourceEntity.ResourceEntityBuilder dummyResourceEntityBuilder(UUID cou
         return ResourceEntity.builder().courseId(courseId).resourceId(resourceId);
     }
 
-    private ResourceDto.Builder dummyResourceDtoBuilder(UUID resourceId, List<CourseIdAvailabilityMapDto> courses){
-        return ResourceDto.builder().setResource_id(resourceId).setCourses(courses);
+    private ResourceDto.Builder dummyResourceDtoBuilder(UUID resourceId, List<UUID> available, List<UUID> unavailable){
+        return ResourceDto.builder().setId(resourceId).setAvailableCourses(available).setUnAvailableCourses(unavailable);
```

Comment:
if you use a method to pass all arguments and then put them in the builder, that defeats the purpose of a builder and you can just call the constructor.
(that's not a todo, just a remark for the future)

Reply:
I agree are the leftovers of a previous refactoring of the representation of the PK for the resource. I'll remove it